.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Is this what we’re gonna do across all our packages?

Reply:
To be honest, I don't know. Some of our projects now include the `package-lock.json` because they use Travis publishing, although the package-lock is not required for that feature.

I was working with @justinbmeyer on `can-cid` and he added the `.npmrc` file to turn off the package lock so I have been following that pattern.

Reply:
Ok, if this is something we’re doing across projects, please file an issue for us to do this with Landscaper.

Reply:
I've been adding this on packages as I touch them also. I've been adding it to `.gitignore` also.

```bash
echo 'package-lock=false' >> .npmrc && echo 'package-lock.json' >> .gitignore
```

Reply:
Created an issue here: https://github.com/canjs/canjs/issues/3622